README.md

```diff
@@ -49,7 +49,9 @@ var isEnabled = normalize(['string', 'boolean'], true);
 // Provide a function as first argument to do custom coercion
 var now = new Date();
 var enabledSince = normalize(function(value) {
-  return value.constructor === Date ? value : null;
+  if (value.constructor === Date) {
```

Comment:
Huh, that's not something I've heard about ternaries before? Anyway, returning `undefined` explicitly is slightly odd. In documentation I think it might help to point out what's expected though, but nitpicking (again).